javaparser-symbol-solver-logic/src/main/java/com/github/javaparser/symbolsolver/logic/TypeDeclarationWithResolutionCapabalities.java

```diff
@@ -0,0 +1,16 @@
+package com.github.javaparser.symbolsolver.logic;
+
+import com.github.javaparser.resolution.declarations.ResolvedMethodDeclaration;
+import com.github.javaparser.resolution.types.ResolvedType;
+import com.github.javaparser.symbolsolver.model.resolution.SymbolReference;
+import com.github.javaparser.symbolsolver.model.resolution.TypeSolver;
+
+import java.util.List;
+
+public interface TypeDeclarationWithResolutionCapabalities {
```

Comment:
Just a minor comment from my side, there's a typo here, the interface should be named "TypeDeclarationWithResolutionCapabilities" ("bil" instead of "bal").